couchbase.py

```diff
@@ -0,0 +1,364 @@
+#!/usr/bin/env python
+# Copyright (C) 2016 SignalFx, Inc.
+
+import json
+import pprint
+import urllib2
+
+import collectd
+import metric_info
+
+# Global constants
+DEFAULT_API_TIMEOUT = 1  # Seconds to wait for the Couchbase API to respond
```

Comment:
I actually just changed this to 60 seconds on rabbitmq because in larger configurations it took longer for the API to respond. It might be worth doing some experiments with couchbase if the API ever takes longer to return, especially on large or busy configs.